pkg/virtctl/expose/BUILD.bazel

```diff
@@ -26,12 +28,14 @@ go_test(
         "expose_suite_test.go",
         "expose_test.go",
     ],
+    x_defs = version_x_defs(),
```

Comment:
What's this exactly?

Reply:
It makes the unit test environment to be loaded with `GitVersion` virtclient properly valued. Without it, the `GitVersion` has the default value that can be found in `kubevirt.io/client-go/version/base.go`. With this value (`v0.0.0-master+$Format:%h$`) the `semver.Make(clientGitVersion)` fails  because it contains some not allowed chars (`$, %`).

Reply:
Cool, thanks!